Add option smartbanner:parent-element (#164)
ain/smartbanner.js: README.md

<meta name="smartbanner:disable-positioning" content="true">
```

#### Define parent element

> Additionally to disabling positioning, you might want to place the smart app banner inside a specific container (e.g. parent element). The parent element is defined by a css selector:

Reviewer: ```suggestion
Additionally to [disabling positioning](#disable-positioning), you might want to place smart app banner inside specific container (e.g. parent element). The parent element is defined by CSS selector:
```

```html
<meta name="smartbanner:disable-positioning" content="true">

> <meta name="smartbanner:parent-element" content="#mobile-navigation.banner-container">

Reviewer: Unrelated to this project, but let's not promote overqualified selectors and go simpler:
```suggestion
<meta name="smartbanner:parent-element" content=".banner-container">
```

```

> If the desired element can't be found by the defined selector the smart app banner will fallback to the document body.

Reviewer: ```suggestion
If targeted container can not be selected, smart app banner will fall back to document body.
```

> 

Reviewer: ```suggestion
```
